docs/tutorials/ruby-driver-create-client.txt

```diff
@@ -739,7 +740,7 @@ URI options are explained in detail in the :manual:`Connection URI reference
    * - compressors=Strings
      - ``:compressors => Array<String>``
 
-       Specified as a comma-separated list. Note that the Ruby driver only supports zlib
+       Specified as a comma-separated list. Note that the Ruby driver only supports zlib and snappy
```

Comment:
This entire paragraph needs to be adjusted since the compression situation has significantly changed. I suggest the following:

Specified as a comma-separated list. The Ruby driver currently supports snappy
and zlib compression algorithms; the first algorithm in the list that the server
also supports will be used. Compression must be explicitly enabled by
specifying the desired compression algorithm; use ``"snappy,zlib"`` for maximum server
compatibility. Snappy compression requires separate installation of the 
``snappy`` Ruby library; if the ``snappy`` Ruby library is not available,
the driver behaves as if snappy compression was not requested.
When using MongoDB 4.0 and earlier, zlib compression is supported by the
server but must be manually enabled; if ``zlib`` is the only compression algorithm
requested (or available due to missing ``snappy`` Ruby library), 4.0 and
earlier servers must be configured to explicitly enable zlib compression.